docs/name-owners/atomic-name-trading/index.md

```diff
@@ -0,0 +1,294 @@
+```
+
+Now Bob can check that everything is fine: 
+
+```
+$ namecoind decoderawtransaction <large hex string>
+{
+  "txid" : "88180e13968cea5ea611fcb34d26b5391bdc9118f8a35fad2495b07be9e88c7b",
+  "version" : 28928,
+  "locktime" : 0,
+  "vin" : [
+    {
+      "txid" : "12ed9f83b83e248e2ca19e0197933914013145981fcacc5ba47c27e60eacbfaa",
+      "vout" : 0,
+      "scriptSig" : {
+        "asm" : "",
+        "hex" : ""
+      },
+      "value" : 974.90000000,
+      "sequence" : 4294967295
+    },
+    {
+      "txid" : "00abd8282558574369038722d24c65355e1b98ca3f00d7d09cb7bea8732b598a",
+      "vout" : 1,
+      "scriptSig" : {
+        "asm" : "",
+        "hex" : ""
+      },
+      "value" : 0.01000000,
+      "sequence" : 4294967295
+    }
+  ],
+  "vout" : [
+    {
+      "value" : 10.00000000,
+      "n" : 0,
+      "scriptPubKey" : {
+        "asm" : "OP_DUP OP_HASH160 3c4b7d4b93bc6194087bbbc422fd6cd1a40f820e OP_EQUALVERIFY OP_CHECKSIG",
+        "hex" : "76a9143c4b7d4b93bc6194087bbbc422fd6cd1a40f820e88ac",
+        "reqSigs" : 1,
+        "type" : "pubkeyhash",
+        "addresses" : [
+          "mm1mGfLEMjFPyGsbkcoYiN6yiPcBztK6Dw"
+        ]
+      }
+    },
+    {
+      "value" : 964.89500000,
+      "n" : 1,
+      "scriptPubKey" : {
+        "asm" : "OP_DUP OP_HASH160 ab96c9bfb7c55de825be7f12bd38dd25b62be02b OP_EQUALVERIFY OP_CHECKSIG",
+        "hex" : "76a914ab96c9bfb7c55de825be7f12bd38dd25b62be02b88ac",
+        "reqSigs" : 1,
+        "type" : "pubkeyhash",
+        "addresses" : [
+          "mwAEUoA3NzSmASyWxrTK1mhsd4pFvruoXj"
+        ]
+      }
+    },
+    {
+      "value" : 0.01000000,
+      "n" : 2,
+      "scriptPubKey" : {
+        "nameOp" : {
+          "op" : "name_update",
+          "name" : "d/my-cool-domain",
+          "value" : "thanks from Sally"
+        },
+        "asm" : "NAME_OPERATION OP_DUP OP_HASH160 ad4a0929e9c7c95910534b93ec0727058a27f2b9 OP_EQUALVERIFY OP_CHECKSIG",
+        "hex" : "5310642f6d792d636f6f6c2d646f6d61696e117468616e6b732066726f6d2053616c6c796d7576a914ad4a0929e9c7c95910534b93ec0727058a27f2b988ac",
+        "reqSigs" : 1,
+        "type" : "pubkeyhash",
+        "addresses" : [
+          "mwKDtLZ9EHSeL8TjkZAcppsnb7M3LRhoGR"
+        ]
+      }
+    }
+  ],
+  "fees" : 0.00500000
+}
+```
+
+The most important pieces to check are highlighted above, but they are obviously the addresses as well as the amounts to be sent. It is also a good idea to check the calculated fees to make sure there's no typo in the amounts resulting in exorbitantly high fees.
+
+When everything looks fine, Bob can sign his inputs to the transaction (after unlocking the wallet if it is password-protected): 
+
+```
+$ namecoind signrawtransaction <large hex string>
```

Comment:
Nit: Now it is `signrawtransactionwithwallet`.

Reply:
Fixed.